documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -132,6 +132,7 @@
 - [ ] Open a pull request against `master` with a merge of the `release/vX.Y.Z` branch. 
    - [ ] Conflict resolution should ignore the changes to `version.go` (keep the `-dev` version from master). 
    - Link to PR: 
+   - Assuming we followed [the process of merging changes into `master` first before backporting to the release branch](https://github.com/filecoin-project/lotus/blob/master/LOTUS_RELEASE_FLOW.md#branch-and-tag-strategy), the only changes should be CHANGELOG updates.  
```

Comment:
if we go with the cherry-pick suggestion above then this line and line 132 need to be changed -- the nice thing about a cherry-pick is that you could isolate a commit to just the changelog and just pull that back in

the downside is that you don't get the assurances of "I've included everything" that you can mostly get from a full merge

@rjan90 what do you think?

Reply:
Makes sense.  Lets decide in https://github.com/filecoin-project/lotus/pull/12416#discussion_r1732038556 and then update accordingly.

Reply:
Addressed in https://github.com/filecoin-project/lotus/pull/12416/commits/7933118228d81ef855425c31fd87f3dc4295c382